iceoryx_posh/include/iceoryx_posh/roudi/port_pool.hpp

```diff
@@ -71,39 +71,39 @@ class PortPool
     cxx::expected<PublisherPortRouDiType::MemberType_t*, PortPoolError>
     addPublisherPort(const capro::ServiceDescription& serviceDescription,
                      mepoo::MemoryManager* const memoryManager,
-                     const ProcessName_t& applicationName,
+                     const RuntimeName_t& applicationName,
```

Comment:
Here the typename and parameter name are a bit out of sync (is an application name always the runtime name?). Can stay this way but maybe it is better to rename it to runtimeName.

Reply:
`applicationName` is misleading, it is not the name of the binary but instead the name of the application's runtime. Fixed